src/libraries/System.IO.Pipelines/tests/PipeWriterTests.cs

```diff
@@ -360,5 +360,27 @@ public async Task GetMemoryFlushWithACompletedReaderNoops()
             Assert.Equal(0, pool.CurrentlyRentedBlocks);
             Assert.Equal(0, Pipe.Writer.UnflushedBytes);
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser), nameof(PlatformDetection.Is64BitProcess))]
+        public void UnflushedBytesShouldOverflowInt()
+        {
+            PipeWriter writer = PipeWriter.Create(Stream.Null);
+            while (writer.UnflushedBytes >= 0 && writer.UnflushedBytes <= int.MaxValue)
+            {
+                Memory<byte> buffer = writer.GetMemory();
+                if (buffer.Length < 47)
+                {
+                    writer.Advance(21);
+                    buffer = writer.GetMemory(47);
+                    writer.Advance(14);
+                }
+                else
+                {
+                    writer.Advance(35);
+                }
+            }
```

Comment:
This is going to need to loop a lot to overflow int. Couldn't we use larger numbers to reduce that? I realize this is copy/paste from the issue, but presumably there's a simpler/faster repro?